### ISSUE TITLE
Error when upgrading AWX from 1.0.2.0 to 1.0.2.289+

### ISSUE THREAD

**Issue by ricalenil** [ansible/awx#884]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - Installer

##### SUMMARY
When upgrading AWX from version 1.0.2.0, it fails with this error in awx_task container.

```
django.db.utils.ProgrammingError: column main_jobtemplate.credential_id does not exist
LINE 1: ...urvey_enabled", "main_jobtemplate"."survey_spec", "main_jobt...
```
                                                             

##### ENVIRONMENT
* AWX version: 1.0.2.0
* AWX install method: docker on linux
* Ansible version: 2.4.2.0
* Operating System: Ubuntu
* Web Browser: firefox 

##### STEPS TO REPRODUCE

- Delete all images and containers.
- I'm keeping pgdata in another dir, not in /tmp
- Run the installer with ansible-playbook -i inventory install.yml 

##### EXPECTED RESULTS
AWX should be upgraded to the latest version without errors.

##### ACTUAL RESULTS

This error in AWX_TASK cointainer.

```
django.db.utils.ProgrammingError: column main_jobtemplate.credential_id does not exist
LINE 1: ...urvey_enabled", "main_jobtemplate"."survey_spec", "main_jobt...
```

##### ADDITIONAL INFORMATION

I'm upgrading from 1.0.1.325 to 1.0.2.0 without errors.

**Comment by wwitzel3** [comment]
See https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/awx-project/PQLxKl5Rj9s/UGy-3VaCCQAJ

**Comment by steaksauce-** [comment]
What the above article means in plain English for those of us who aren't familiar with how to set things up from the default:
modify the inventory file in the installer directory:
Change line
`dockerhub_version=latest`
to
`dockerhub_version=1.0.2.0`
  
Edit: fixed the file as per @shortsteps comment below -- thanks!

**Comment by shortsteps** [comment]
@steaksauce- great job to explain that, but maybe you meant the `inventory` file not install.yml.
Sorry if I correct you but it's just to avoid future confusion :)
